packages/gatsby-telemetry/src/__tests__/error-helpers.ts

```diff
@@ -86,11 +86,16 @@ describe(`Errors Helpers`, () => {
 
       const sanitizedErrorString = sanitizeError(e, `/`)
 
-      expect(sanitizedErrorString.includes(errormessage)).toBe(true)
+      expect(sanitizedErrorString.includes(errorMessage)).toBe(true)
       expect(sanitizedErrorString).toEqual(
         expect.not.stringContaining(`sidharthachatterjee`)
       )
-      expect(sanitizedErrorString.match(/\$SNIP/g).length).toBe(4)
+      const result = sanitizedErrorString.match(/\$SNIP/g)
+      expect(result).toBeTruthy()
+      if (result === null) {
```

Comment:
This feels stupid, maybe we should not be this strict for tests...

Reply:
Isn't this redundant? If `result` is null, the toBeTruthy will fail, yeah?

Reply:
yes, but not with the current linting rules. TS isn't aware of the type verifications jest provides, so even if `toBeTruthy` passes, typescript isn't certain that the result is non null.

If I would remove the null check the typecheck will fail with the following error:
```
packages/gatsby-telemetry/src/__tests__/error-helpers.ts:95:14 - error TS2531: Object is possibly 'null'.

95       expect(result.length).toBe(4)
                ~~~~~~
```

A nicer solution would be to have less strict ts-linting for tests as a "NullPointerException" would be an acceptable test failure IMO

Reply:
Ahhhhhhh got it. I should probably pull this down, but I think something like this would work.

```ts
const result = sanitizedErrorString.match(/\$SNIP/g) as string[]
```

but sanity check me 🙃 I think updating code for TypeScript correctness is super weird.

Reply:
yea, that will work too.